behave_django/management/commands/behave.py

```diff
@@ -127,6 +137,13 @@ def handle(self, *args, **options):
                 ' together with --use-existing-database'
             ))
 
+        both_active = options['use_existing_database'] or options['simple']
+        if options['runner_class'] and both_active:
```

Comment:
We could check for a deviation from the default value instead of just `options['runner_class']`, e.g.

```suggestion
        if options['runner_class'] != DEFAULT_RUNNER and both_active:
```

Then again, do the switches `--use-existing-database` and `--simple` (always or at all) make sense with a custom runner? We might make those switches mutually exclusive, otherwise. Would that be a sane compromise?

Reply:
Just committed, see what you think

Reply:
Yup. On it ;)